src/containers/Board/Cards/Card.js

```diff
@@ -8,10 +8,24 @@ const propTypes = {
 const galPng = require('../../../assets/images/gal.png');
 const delPng = require('../../../assets/images/del.png');
 
+function timeformat(timee) {
+  let ftime = "";
+  let past = false;
+  if (timee < 0) {timee = 0-timee; past = true}
+  if (timee/86400000 > 1) {ftime += Math.floor(timee/86400000) + 'd';};
+  let ttime = timee%86400000;
+  if (ttime/3600000 > 1) {ftime += Math.floor(ttime/3600000) + 'h';};
+  ttime = ttime%3600000;
+  if (ttime/60000 > 1) {ftime += Math.floor(ttime/60000) + 'm';};
+  ttime = ttime%60000;
+  ftime += Math.floor(ttime/1000) + 's '+ ["until","past"][Number(past)] + ' deadline';
+  return ftime;
+};
+function test(tests) {return tests}
 
 const Card = (props) => {
   const { style, item } = props;
-
+  let styledate = {background: ['#ff8888','#ffff88','#88ff88'][Number(item.timedue > Date.now()) + Number(item.timedue - 86400000 > Date.now())]}
```

Comment:
The hex codes could be pulled out as constants and named: e.g. `OVERDUE_COLOUR = #88ff88`

Reply:
Also the line could be written so it is more readable with functions that are named so the reader understands what is going on, looking at the logic. This is also a good function to unit test.